public/config.default/theme/picoman.php

```diff
@@ -0,0 +1,230 @@
+	unsures.forEach(el => {
+		var currNo = document.getElementById("qNum").textContent-1;
+		if(currNo===el){
+			unsureBtn.setAttribute("onclick","removeUnsure()");
+			unsureBtn.style.backgroundColor = "#fff176";
+			unsureBtn.style.color = "#575757";
+			unsureBtn.style.borderColor = "#08769b";
+			
+			qNumSpan.style.backgroundColor = "#fff176";
+			qNumSpan.style.color = "#575757";
+			qNumSpan.style.borderColor = "#08769b";
+		}
+	  setUnsureLiBg(el);
+	});
+}
+
+function addUnsure(){
+	if(fTestForm){
+		var index = document.getElementById("qNum").textContent-1;
+		var unsureBtn = document.getElementById("unsure");
+		unsureBtn.setAttribute("onclick","removeUnsure()");
+		
+		unsureBtn.style.backgroundColor = "#fff176";
+		unsureBtn.style.color = "#575757";
+		unsureBtn.style.borderColor = "#08769b";
+		
+		qNumSpan.style.backgroundColor = "#fff176";
+		qNumSpan.style.color = "#575757";
+		qNumSpan.style.borderColor = "#08769b";
+		
+		setUnsureLiBg(index);
+		let unsure;		
+		if(localStorage.getItem('unsure') === null){
+			unsure = [];
+		}else{
+			unsure = JSON.parse(localStorage.getItem('unsure'));
+		}
+		unsure.push(index);
+		localStorage.setItem('unsure', JSON.stringify(unsure));
+	}
+}
+
+function removeUnsure(){
+	if(fTestForm){
+		var index = document.getElementById("qNum").textContent-1;
+		var unsureBtn = document.getElementById("unsure");
+		unsureBtn.setAttribute("onclick","addUnsure()");
+		unsureBtn.removeAttribute("style");
+		qNumSpan.removeAttribute("style");
+		document.querySelectorAll("ol.qlist li")[index].removeAttribute("style");
+		let unsure;
+		if(localStorage.getItem('unsure') === null){
+			unsure = [];
+		}else{
+			unsure = JSON.parse(localStorage.getItem('unsure'));
+		}
+		for(var i = 0; i < unsure.length;){
+			if(unsure[i] === index){
+				unsure.splice(i, 1);
+			}else{
+				i++;
+			}
+		}
+		localStorage.setItem('unsure', JSON.stringify(unsure));
+	}
+}
+/*
+* Please don't remove or alter this credit, keep visible and reachable, contact me if you want to make donation. email: mamansulaeman86@gmail.com telegram: @mamans86 
+*/
+document.querySelector("body").insertAdjacentHTML("beforeend", "<div style='padding:0 1rem 1rem 1rem;text-align:center;background:var(--contrast);color:var(--background)'><span style='font-size:0.85rem'><a style='color:var(--color-warning)' href='https://mamans86.blogspot.com' target='blank'>PicoMan Theme</a> created with love by <a style='color:var(--color-warning)' href='https://mamans86.blogspot.com' target='blank'>mamans86</a></span></div>");
+document.getElementById("scrollayer").insertAdjacentHTML("beforeend", "<div style='position:fixed;bottom:1rem;right:1rem'><span style='font-size:0.75rem'><a href='https://mamans86.blogspot.com' target='blank'>TCExam PicoMan Theme</a></span></div>");
+/*
+* Please don't remove or alter this credit, keep visible and reachable, contact me if you want to make donation. email: mamansulaeman86@gmail.com telegram: @mamans86 
+*/
```

Comment:
Credits should go in:
https://github.com/tecnickcom/tcexam/blob/develop/admin/code/tce_page_info.php
Active links should point only to the official TCExam website.
Please refer and sign:
https://cla-assistant.io/tecnickcom/tcexam?pullRequest=350

Reply:
Btw, this script doesn't modify current links. The current link in bottom of page is still exist, visible and point to tcexam.org and tecnick.com

Cheers